src/pages/Admin/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -43,7 +43,7 @@ const ToggleElement: React.FC<ToggleElementProps> = ({
     <S.ItemBody>
       <Checkbox
         name={name}
-        forwardRef={forwardRef}
+        forwardedRef={forwardRef}
         defaultChecked={isChecked}
         toggler
       />
```

Comment:
When the changes called out elsewhere in this review to properly fix this to use the `forwardRef` function are implemented, then I believe this prop will change to just being `ref` here.